### ISSUE TITLE
Icon box cannot be aligned

### ISSUE THREAD

**Issue by quantumleap33** [ThemeFuse/Brizy#195]
When you add an icon box, it aligns to the left by default but there is no way to align the whole icon box to the center or the right and you have to use padding tricks to make it work. It would be nice if this was fixed so that you can get the whole icon box to aligned.

https://youtu.be/cetWg1h26iE?t=623

**Comment by bagrinsergiu** [comment]
Hi 

I understand what you mean. At some point we had this feature in Icon Text but we disabled it. 

I don't like Bundle Elements or All in one elements. In future we plan to add a Symbol section where we can create your own symbol ( 2-3 elements in one Symbol ) very similar to Sketch Symbols https://sketchapp.com/docs/symbols/

P.S. maybe we will add this feature againfor Icon Text 

Thanks for suggestion.
Sergiu

**Comment by quantumleap33** [comment]
Yes, I know what you mean, I don't like bundled elements either. Yet again the icon box IS a bundled element and should be treated as such. Thank you!
